starknet_py/net/udc_deployer/deployer.py

```diff
@@ -64,20 +64,52 @@ async def prepare_contract_deployment(
         :param calldata: Constructor args of the contract to be deployed
         :param max_fee: Max amount of Wei to be paid when executing transaction
         :param auto_estimate: Use automatic fee estimation, not recommend as it may lead to high costs
-        :return: InvokeFunction
+        :return: NamedTuple with invoke_tx and address fields
```

Comment:
Maybe let's make a docstring a little bit more descriptive, so it says it's the address of a contract that will be deployed or something like that.